pkgs/by-name/ce/celeste64/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildDotnetModule,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  SDL2,
+  libGL,
+  mesa,
+  fmodex,
+  libdecor,
+  libxkbcommon,
+  wayland,
+  systemd,
+  alsa-lib,
+  libselinux,
+  libX11,
+  libXfixes,
+  libXext,
+  libXi,
+  libXrandr,
+  libXcursor,
+  withSELinux ? false,
+}:
+buildDotnetModule {
+  pname = "celeste64";
+  version = "0-unstable-2024-01-31";
+
+  src = fetchFromGitHub {
+    repo = "Celeste64";
+    owner = "ExOK";
+    sha256 = "sha256-S9KOtCrJSsIQxvZWjiHUYEiM9/TbO0q6vSDPwJfyW3I=";
+    rev = "62417823a30582e8561075fe0f7d4b398aedc353";
+  };
+  projectFile = "Celeste64.csproj";
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+  nugetDeps = ./deps.nix;
+  strictDeps = true;
+  runtimeDeps =
+    [
+      mesa
+      libdecor
+      libGL
+      SDL2
+      fmodex
+      libxkbcommon
+      systemd
+      alsa-lib
+      wayland
+      libX11
+      libXfixes
+      libXext
+      libXcursor
+      libXi
+      libXrandr
+    ]
+    ++ lib.optionals withSELinux [ libselinux ];
+
+  postFixup = ''
+    cp -r $src/Content $out/lib/$pname/
+  '';
+
+  meta = {
+    license = with lib.licenses; [ unfree mit ];
+    platforms = lib.platforms.linux;
```

Comment:
Does the game support non-x64 architectures? Maybe this should be `[ "x86_64-linux" ]`. The [project file][1] only lists x64 as a supported platform. Maybe this will change in the future, since [macOS support is planned][2].

[1]: https://github.com/ExOK/Celeste64/blob/62417823a30582e8561075fe0f7d4b398aedc353/Celeste64.csproj#L12
[2]: https://mastodon.exok.com/@EXOK/111842053371358280

Reply:
Will use that, thank you.